src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -154,9 +154,12 @@ class WorkspaceMembersPage extends React.Component {
 
     /**
      * Add or remove all users from the selectedEmployees list
+     * @param {Object} memberList
      */
-    toggleAllUsers() {
-        let policyMemberList = lodashGet(this.props, 'policyMemberList', {});
+    toggleAllUsers(memberList) {
+        // If a memberList is provided, only select those members. Otherwise, select all the policy ones
+        let policyMemberList = _.isEmpty(memberList) ? lodashGet(this.props, 'policyMemberList', {}) : memberList;
+
         policyMemberList = _.filter(_.keys(policyMemberList), policyMember => policyMemberList[policyMember].pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE);
         const removableMembers = _.without(policyMemberList, this.props.session.email, this.props.policy.owner);
```

Comment:
Shouldn't we remove this because we already only `removableMembers` in `memberList` list

Reply:
I left it as a sanity check to make sure no owners were incorrectly selected/unselected, but we can remove it and leave it only if the memberList is empty, if you feel strongly about it

Reply:
We can remove this, this method is used in one place, which has `memberList` that is removable only. All these checks are taken care of there.

<img width="1323" alt="Screenshot 2023-03-13 at 10 17 03 PM" src="https://user-images.githubusercontent.com/85645967/224769537-bff87bf3-27b3-4c95-8ce0-0f66f1e6508e.png">